doc/manuals/sprokit/distributed_pipeline.rst

```diff
@@ -187,7 +205,13 @@ data_serializer algorithms can be displayed with the following command
 
 
 Transport Processes
-===================
+-------------------
+
+KWIVER’s transport strategy is structured as Sprokit end-caps (the needs and
+requirements for the transport implementations are somewhat dependent on
+Sprokit’s stream processing implementation and so don’t lend themselves to
+implementation as KWIVER arrows). The current implementation focuses on a
```

Comment:
The sentence that starts with ``The current implementation...`` reads a little odd to me.  Would either of the following sound better?
- ``The current implementation focuses on one-to-many and many-to-one topologies for VITAL data types.``
- ``The current implementation focuses on one-to-many and many-to-one topologies for any one VITAL data type.``